packages/framework/src/Publications/Views/Components/RelatedPublicationsComponent.php

```diff
@@ -25,72 +29,101 @@ public function render(): Factory|View
         return view('hyde-publications::components.related-publications');
     }
 
-    protected function makeRelatedPublications(int $max = 5): array
+    protected function makeRelatedPublications(int $max = 5): Collection
     {
         // Get current publicationType
-        $currPage = Hyde::currentRoute()->getPage();
-        $publicationType = $currPage->getType();
+        $currHydePage = Hyde::currentRoute()->getPage();
```

Comment:
Please see if you can expand all the abbreviated variable names.

Reply:
Done!